plugins/scaffolder-backend/src/scaffolder/tasks/convertZodtoJson.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { z } from 'zod';
+import zodToJsonSchema from 'zod-to-json-schema';
+
+/** @public */
+export function convertZodtoJson(zodSchema: z.ZodType<any, any, {}>) {
```

Comment:
I'm not sure we need so much code here, I think that we want to copy some of the code from the usage in the frontend: 
https://github.com/backstage/backstage/blob/da25293910f208d00296af7c7de497a8f8ff7278/plugins/scaffolder/src/components/fields/utils.ts#L35-L62 I think should help us get on the way.